src/backend/dx11/src/debug.rs

```diff
@@ -0,0 +1,101 @@
+use winapi::um::d3d11;
+
+use wio::com::ComPtr;
+use wio::wide::ToWide;
+
+use std::ffi::OsStr;
+use std::env;
+
+// TODO: replace with new winapi version when available
+#[allow(bad_style, unused)]
+mod temp {
+    use winapi::shared::minwindef::{BOOL, INT}; 
+    use winapi::um::unknwnbase::{IUnknown, IUnknownVtbl}; 
+    use winapi::um::winnt::LPCWSTR; 
+
+    RIDL!{#[uuid(0xb2daad8b, 0x03d4, 0x4dbf, 0x95, 0xeb, 0x32, 0xab, 0x4b, 0x63, 0xd0, 0xab)]
+    interface ID3DUserDefinedAnnotation(ID3DUserDefinedAnnotationVtbl):
+        IUnknown(IUnknownVtbl) {
+        fn BeginEvent(
+            Name: LPCWSTR,
```

Comment:
any reason those are split into multiple lines?

Reply:
Taken from winapi, which has some unorthodox formatting. Hopefully there'll be a new winapi release around next week